tests/queries/0_stateless/03668_parquet_min_bytes_for_seek_zero.reference

```diff
@@ -0,0 +1,2 @@
+1
+
```

Comment:
The test fails because of this empty line, I think: https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=88784&sha=f37701f26abe71ec4744fbda0ce7f12a9ec1ee03&name_0=PR